cmake/projects/spdlog/hunter.cmake

```diff
@@ -73,7 +73,18 @@ hunter_add_version(
     "https://github.com/hunter-packages/spdlog/archive/v0.16.3-p1.tar.gz"
     SHA1
     1400ca456ad7c5ba89491f0dd3cd8935a16fe65c
-)  
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    spdlog
+    VERSION
+    "1.2.1"
+    URL
+    "https://github.com/gabime/spdlog/archive/v1.2.1.tar.gz"
```

Comment:
There are some patches needed to adapt upstream version for Hunter, e.g. use Hunter to install "fmt" library: https://github.com/hunter-packages/spdlog/blob/hunter-0.16.3/CMakeLists.txt#L49-L52

I've created branch `hunter-1.2.1` based on upstream 1.2.1 tag:
* https://github.com/hunter-packages/spdlog/tree/hunter-1.2.1

Please send there pull request with patches from previous `hunter-0.6.13` branch:
* https://github.com/hunter-packages/spdlog/tree/hunter-0.16.3

Reply:
The upstream bundled `fmt` library. https://github.com/gabime/spdlog/tree/v1.2.1/include/spdlog/fmt Hunterized package is not needed anymore.

Reply:
Package manager should be responsible for managing dependencies, not package itself. In general the usage of bundled dependencies is a dirty hack, not a solution.

* https://docs.hunter.sh/en/latest/faq/why-do-we-need-forks.html#solution-for-bundled-sources
* https://cgold.readthedocs.io/en/latest/tutorials/install/managing-dependencies.html
* https://docs.hunter.sh/en/latest/creating-new/create/cmake-dependencies.html
* https://wiki.gentoo.org/wiki/Why_not_bundle_dependencies

Reply:
It makes sense. https://github.com/hunter-packages/spdlog/pull/7. I can help package it and update this PR with new hunterized spdlog.